### PR TITLE
Chrome Gamepad patch

### PR DESCRIPTION
**Pull request by nerdgilbert**
## Major Changes
- [x] Fix for: https://github.com/OpenROV/openrov-software/issues/626

### CODE REVIEW

File: src/static/js/libs/gamepad.js [hunk 1]
```diff
@@ -475,18 +484,53 @@
...
+
   /**
  * Updates the controllers, triggering TICK events.
  */
   Gamepad.prototype._update = function () {
     var self = this, controlName, isDown, lastDown, downBtnIndex, mapping, value, i, j;
+
+    //Support chrome, scan for gamepads
+    this._scanGamepads();
     switch (this.platform) {
```
Comment (BrianAdams) [inline src/static/js/libs/gamepad.js]: I believe the entire switch statement can be removed as the _scanGamepads() that you added duplicates the same feature of polling the gamepad API for available gamepads.

**Comment by nerdgilbert** [comment]
I remember you mentioning that and I was curious because your changes exist
in the WebKit update function, no?

**Comment by BrianAdams** [comment]
@nerdgilbert , If I recall the flow correctly, the function you added does exactly what the switch statement does.  Leaving the switch statement essentially does another poll looking for gamepad devices.

**Comment by nerdgilbert** [comment]
Verified on hardware and MOCK.

**Pull request merged**